### ISSUE TITLE
Trained model but output has horizontal lines

### ISSUE THREAD

**Issue by nickmen** [jcjohnson/fast-neural-style#135]
Hello, I tried to train today with following command but the output has the following noticeable horizontal line. I tried to use different input as style image and notice the horizontal line too. How could i generate a more painting like output? I have already set style weight to 28 with 40000 iteration. 
What could be the issue? Appreciate any help. 

Using:
Vgg16, COCO 2014 data

Command: (other option i leave it as default)
th train.lua -h5_file file.h5  -style_image images/styles/style.jpg -style_image_size 350 -content_weights 1.0 -style_weights 28.0 -checkpoint_name style_portrait -gpu 0

Style:
![style](https://user-images.githubusercontent.com/24219966/28808724-eb76dcec-76b0-11e7-83df-192dac1c0a12.jpg)

Input:
![input](https://user-images.githubusercontent.com/24219966/28808748-14e915b8-76b1-11e7-8342-b0790704050a.jpg)

Output:
![output](https://user-images.githubusercontent.com/24219966/28808752-19a2d8be-76b1-11e7-95a4-3726c41acbf1.png)

**Comment by hongzhongfang** [comment]
hello @nickmen 
I have some trouble in training new models.
Can you describe the training process in detail? especiall the step1: prepare a dataset.
thanks

**Comment by alexiscreuzot** [comment]
I do have the same issue, I wonder if this is not something related to the sizes

**Comment by Bella31** [comment]
I also for some style models have the same issue, not sure why sometimes it works and sometimes it doesn't. Is there a way to quantify if a model is successful (maybe by checking the loss during training)?

**Comment by alexiscreuzot** [comment]
I actually went with PyTorch instead of Torch and it gives me better results on top of being easier to use...

**Comment by oleprinds** [comment]
Adding an upscale pass to the -arch option helped remove this pattern for me..

E.g -arch c9s1-16,d32,d64,R64,R64,R64,R64,R64,U2,c3s1-32,U2,c9s1-3

See #37
